presto-orc/src/main/java/com/facebook/presto/orc/reader/ByteSelectiveStreamReader.java

```diff
@@ -175,6 +182,59 @@ else if (filter == null) {
     private int readWithFilter(int[] positions, int positionCount)
             throws IOException
     {
+        int totalPositionCount = positions[positionCount - 1] + 1;
```

Comment:
I don't see new tests to cover new code paths. Are they all covered by the existing tests?

Reply:
Yes. They are already covered in `testByteValues()`, `testByteValuesRepeat()`, `testByteValuesPatchedBase()` and `testBooleanSequence()`.  Skipping will be covered by new tests mentioned above in `testArrays()` for boolean and byte types.